tests/testsuite/install.rs

```diff
@@ -2780,3 +2780,85 @@ fn dry_run_incompatible_package() {
         .run();
     assert_has_not_installed_exe(paths::cargo_home(), "some-package-from-the-distant-future");
 }
+
+#[cargo_test]
+fn dry_run_upgrade() {
```

Comment:
Can these tests be moved to the original test commit?

Reply:
Done